js/index.js

```diff
@@ -1,23 +1,37 @@
 // ITERATION 1
 
 function updateSubtotal(product) {
-  console.log('Calculating subtotal, yey!');
-
-  //... your code goes here
+    console.log('Calculating subtotal, yey!');
+
+  
+  const price = +product.querySelector ('.price span').innerHTML;
```

Comment:
```suggestion
  const price = +product.querySelector ('.price span').innerText;
```
Siempre que puedas evitar usar el HTML, mejor!!!